components/amorphic/lib/AmorphicServer.ts

```diff
@@ -61,7 +61,11 @@ export class AmorphicServer {
         const amorphicOptions = AmorphicContext.amorphicOptions;
         const mainApp = amorphicOptions.mainApp;
         const appConfig = AmorphicContext.applicationConfig[mainApp];
-        const server = new AmorphicServer(express(), appConfig.appConfig.serverMode);
```

Comment:
Thats why they should use startPersistorMode for serverless because that doesn't call the createServer. Its just here so that if they do use listen it won't create the server anyways but there will be a log that if they use serverless they should use startPersistorMode instead.